src/super_gradients/common/crash_handler/crash_tips_setup.py

```diff
@@ -0,0 +1,24 @@
+import os
+import atexit
+
+from super_gradients.common.crash_handler.exception import ExceptionInfo
+from super_gradients.common.abstractions.abstract_logger import get_logger
+
+logger = get_logger(__name__)
+
+
+def crash_tip_handler():
+    """Display a crash tip if an error was raised"""
+    crash_tip_message = ExceptionInfo.get_crash_tip_message()
+    if crash_tip_message:
+        print(crash_tip_message)
+
+
+def setup_crash_tips() -> bool:
+    if os.getenv("CRASH_HANDLER", "TRUE") != "False":
```

Comment:
you are checking if the string value is "False" but in your message you  say "set your environment variable to CRASH_HANDLER=FALSE" 
will this work?

Reply:
No it's a mistake, I fixed it